src/main/java/com/pi4j/driver/interfaces/Video.java

```diff
@@ -0,0 +1,12 @@
+package com.pi4j.driver.interfaces;
+
+import java.awt.image.BufferedImage;
+
+public interface Video {
```

Comment:
As per the readme of the repo we don't yet want to use interfaces. I think as @stefanhaustein wrote, just create a St7789 class with the public methods required to use the display. In a later stage when multiple such devices have been added, we can think about extracting interfaces which will work across multiple devices.